playwright/tests/customers.spec.ts

```diff
@@ -0,0 +1,64 @@
+import { CustomersPage } from "@pages/customersPage";
+import { test } from "@playwright/test";
+import * as faker from "faker";
+import { CUSTOMERS, CUSTOMER_ADDRESS } from "@data/e2eTestData";
+
+test.use({ storageState: "playwright/.auth/admin.json" });
+let customersPage: CustomersPage;
+
+test.beforeEach(({ page }) => {
+  customersPage = new CustomersPage(page);
+});
+
+test("TC: SALEOR_119 Create customer @e2e @customer", async () => {
+  const firstName = faker.name.firstName();
+  const lastName = faker.name.lastName();
+  const email = faker.internet.email();
+  const note = faker.lorem.sentence();
+  await customersPage.goToCustomersListView();
+  await customersPage.clickOnCreateCustomer();
+  await customersPage.fillFirstAndLastName(firstName, lastName);
+  await customersPage.fillEmail(email);
+  await customersPage.fillAddress(CUSTOMER_ADDRESS.createCustomerAddress);
+  await customersPage.fillNote(note);
+  await customersPage.saveCustomer();
+  await customersPage.successBanner.waitFor({ state: "visible" });
+});
+test("TC: SALEOR_120 Delete customer @e2e @customer", async () => {
+  await customersPage.gotoCustomerDetailsPage(CUSTOMERS.deleteCustomer.id);
+  await customersPage.deleteCustomer();
+  await customersPage.deleteDialog.clickDeleteButton();
+  await customersPage.successBanner.waitFor({ state: "visible" });
```

Comment:
I would add more assertions to your tests as relying only on the success banner doesn't necessarily mean that all intended data has been saved.

Reply:
I agree. I guess there is a redirect to the list view so maybe we could use it in the assertion, if not use more specific assertion based on what happens

Reply:
I added `expect(getCustomerRequest.status()).toEqual(404)`, i was much easier to get this from api, then go to customer list, and then go through few pages to make sure that this customer does not exist

Reply:
Good idea, sorry didn't notice!